README.md

```diff
@@ -82,6 +82,13 @@ the VM, possibly with extra options for tracing and halting on test failures
 
     TEST_OS=centos-8-stream test/check-application -tvs
 
+It is possible to setup the test environment without running the tests:
+
+    export TEST_OS=centos-8-stream
+    test/run prepare
```

Comment:
We usually write our docs with setting the env just for the single command. If users know what they are doing, they will know `export`, but we don't recommend that -- one tends to forget, and it's rather confusing.